config.js

```diff
@@ -624,6 +624,7 @@ config.AWS_LOG_CANDIDATES_LIMIT = 10;
 config.BUCKET_LOG_REPLICATOR_DELAY = 5 * 60 * 1000;
 config.AZURE_QUERY_TRUNCATION_MAX_SIZE_IN_BITS = 10 * 1024 * 1024;
 config.BUCKET_DIFF_FOR_REPLICATION = true;
+config.BUCKET_DIFF_FOR_DELETION = false;
```

Comment:
This should not be consumed via the config file

Reply:
Thanks, updated the code.